### PR TITLE
Add rule 5-2

### PR DESCRIPTION
**Pull request by xingcx**

### CODE REVIEW

File: docs/section5/Rule5-1-2.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- For each building segment in the Baseline model: ```for building_segment_b in B_RMR.building.building_segments:```  
+
+  - For each thermal_block in building segment: ```thermal_block_b in building_segment_b.thermal_blocks:```  
+  
+    - For each zone in thermal block: ```zone_b in thermal_block_b.zones:```  
+
+      - For each space in zone: ```space_b in zone_b.spaces:```  
+
+        - For each surface in space: ```for surface_b in space_b.surfaces:```  
+
```
Comment (supriyagoel) [inline docs/section5/Rule5-1-2.md:28]: @xingcx, @JasonGlazer the RMR will be changed to have surfaces at zone level. the RDS will need to be updated after that happens.

File: docs/section5/Rule5-1-2.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+          - Check that surface is exterior: ```if ( surface_b.adjacent_to == "EXTERIOR" ): exterior_vertical_surface_b = surface_b```  
+
+          **Rule Assertion:** Baseline exterior surface does not cast shade:  
+
+          - Case 1: ```exterior_vertical_surface_b.does_cast_shade == FALSE: PASS```  
+
```
Comment (supriyagoel) [inline docs/section5/Rule5-1-2.md:34]: @JasonGlazer and @mkarpman - i see that E+ has a boolean input for whether a surface is self-shading- which could be used to populate this input. However, do you know if other tools (like eQuest ) have a flag for this as well?

**Review by supriyagoel — APPROVED** [review]

**Review by mkarpman — APPROVED** [review]

File: docs/section5/Rule5-1-2.md (hunk 2 above)
Comment (mkarpman) [reply docs/section5/Rule5-1-2.md:32]: There is one in eQUEST (shading-surface = yes/no)

File: docs/section5/Rule5-1-2.md (hunk 1 above)
Comment (mkarpman) [reply docs/section5/Rule5-1-2.md:26]: Agreed. Everything else looks good to me.

**Pull request merged**